components/UserLoginCard/index.js

```diff
@@ -0,0 +1,67 @@
+import Flex from "../Layout/Flex";
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+import useUser from "@/hooks/useUser";
+import { useState } from "react";
+import { useRouter } from "next/router";
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledImageWrapper = styled.div`
+  height: 200px;
+  width: 200px;
+  border-radius: 50%;
+  overflow: hidden;
+  position: relative;
+`;
+
+const StyledStarshipImage = styled(Image)`
+  object-fit: cover;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+`;
+
+const StyledParagraph = styled.p`
+  color: var(--secondary-color);
+  text-align: center;
+  font-size: 2rem;
+`;
+
+const StyledButton = styled.button`
+  background-color: Transparent;
+  background-repeat: no-repeat;
```

Comment:
I have the feeling that you do not need this 🤔 🧹 
```suggestion

```

Reply:
We changed that StyledButton on the branch Berna and Ping are working on right now! :)